client/src/components/global/__tests__/Date-test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import Date from '../Date';
+
+function setup() {
+  let props = {
+    prefix: "test",
+    format: "MMMM, YYYY",
+    date: '01-01-2017'
+  };
+
+  let output = TestUtils.createRenderer().render(<Date {...props} />);
+  return {
+    props,
+    output
+  };
+}
+
+describe('components', () => {
+  describe('Date', () => {
+    it('renders the date with correct prefix', () => {
+      const { output } = setup();
+      expect(output.type).toBe('span');
+      expect(output.props.children).toEqual('test January, 2017');
+    });
+  });
+});
```

Comment:
I'd like to see more tests here. What happens when the component is not passed a `date` prop? Does it work? Does it render nothing? What happens when it is not passed a format? Or when it is not passed a prefix? More tests would better illustrate the component's ability to work with different sets of props.